src/lib/swapper/swappers/ThorchainSwapper/utils/getLongtailQuote.ts

```diff
@@ -70,11 +70,8 @@ export const getLongtailToL1Quote = async (
   const publicClient = viemClientByChainId[sellChainId as EvmChainId]
   assert(publicClient !== undefined, `no public client found for chainId '${sellChainId}'`)
 
-  const poolAddresses: Map<Address, FeeAmount> = generateV3PoolAddressesAcrossFeeRange(
-    POOL_FACTORY_CONTRACT_ADDRESS,
-    tokenA,
-    tokenB,
-  )
+  const poolAddresses: Map<Address, { token0: Address; token1: Address; fee: FeeAmount }> =
```

Comment:
ditto `address` terminology on `token0` and `token`

Reply:
https://github.com/shapeshift/web/pull/5902/commits/4bde8e1a0bb1460511b5d4254c174975d04efeb4